dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/net/UploadStatus.kt

```diff
@@ -18,34 +18,36 @@ internal enum class UploadStatus {
     HTTP_SERVER_ERROR,
     UNKNOWN_ERROR;
 
-    fun logStatus(context: String) {
+    fun logStatus(context: String, byteSize: Int) {
         when (this) {
             NETWORK_ERROR -> devLogger.e(
-                "Unable to send batch ($context) because of a network error; " +
-                    "we will retry later."
+                "Unable to send batch with byteSize: $byteSize ($context)" +
```

Comment:
with byteSize is weirdly phrased for a user targeted message. I'd use 
"… batch [$byteSize bytes] ($context)…"

Reply:
Yeah...this is better, was not really sure of that message either 👍